lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -54,16 +54,16 @@ int convertButtonToButtons(int button) {
 
 class PointerBinding {
   /// The singleton instance of this object.
-  static PointerBinding get instance => _instance;
-  static PointerBinding _instance;
+  static PointerBinding? get instance => _instance;
+  static PointerBinding? _instance;
 
-  static void initInstance(html.Element glassPaneElement) {
+  static void initInstance(html.Element? glassPaneElement) {
```

Comment:
`glassPaneElement` should never be null.

Reply:
Done.